src/main/java/at/hannibal2/skyhanni/config/features/garden/GardenConfig.java

```diff
@@ -16,6 +18,10 @@
 
 public class GardenConfig {
 
+    @Expose
+    @Category(name = "Lane Switching", desc = "Settings for Lane Switches")
+    public LaneswitchConfig laneswitch = new LaneswitchConfig();
+
```

Comment:
I don't know if this single features should be a top level category. Maybe move this into the optimal speed category (although that might need a better name then). Alternatively this could just be in the normal garden config. Also this probably shouldn't be the first sub category.

Reply:
Make it an accordion instead of a category and put it between the optimalSpeeds and gardenLevels (Just a guess that it might fit there well)